packages/valory/skills/abstract_round_abci/tests/test_handlers.py

```diff
@@ -407,6 +407,7 @@ def test_handle_request(self, caplog: LogCaptureFixture) -> None:
             assert log_message in caplog.text
 
     # response
+    @pytest.mark.skip  # Logic needs updating
```

Comment:
Please fix the test

Reply:
Resolved in https://github.com/valory-xyz/open-autonomy/pull/1448